acvm_backend_barretenberg/Cargo.toml

```diff
@@ -22,8 +28,16 @@ blake2 = "0.9.1"
 tempfile = "*"
 
 [features]
-default = [
-    # "acvm/bn254",
+default = ["native"]
+
+native=[
+    "dep:barretenberg-sys",
```

Comment:
why do you need "dep:"?

Reply:
Some discussion of this above: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies